modules/pubCommonIdSystem.js

```diff
@@ -46,52 +140,78 @@ export const pubCommonIdSubmodule = {
    * decode the stored id value for passing to bid requests
    * @function
    * @param {string} value
+   * @param {SubmoduleConfig} config
    * @returns {{pubcid:string}}
    */
-  decode(value) {
-    return { 'pubcid': value }
+  decode(value, config) {
+    const sharedId = readData(config);
+    const idObj = {'pubcid': value};
+    if (sharedId) idObj['sharedid'] = {id: sharedId};
+    return idObj;
   },
   /**
    * performs action to obtain id
    * @function
    * @param {SubmoduleConfig} [config]
    * @returns {IdResponse}
    */
-  getId: function ({params: {create = true, pixelUrl} = {}} = {}) {
+  getId: function (config = {}) {
+    const {params: {create = true, pixelUrl} = {}} = config;
```

Comment:
where does this config get set, by the publisher? i don't see any docs about setting `create` or `pixelUrl` here https://docs.prebid.org/dev-docs/modules/userId.html#pubcommon-id ... should those be added?

Reply:
Please see https://github.com/prebid/prebid.github.io/pull/2417